Makefile

```diff
@@ -0,0 +1,59 @@
+SHELL        = /bin/sh
+
+CARGO        := cargo
+FETCHFLAGS   := --locked
+CARGOFLAGS   := --release --locked --offline --no-default-features
+ASCIIDOCTOR  := asciidoctor
+MANFLAGS     := -b manpage -D build
+HTMLFLAGS    := -D build -a toc=left -a toclevels=3 -a sectlinks
+INSTALL      := install
+
+prefix       := /usr/local
+DESTDIR      := $(prefix)
+bindir       := $(DESTDIR)/bin
+datarootdir  := $(DESTDIR)/share
+datadir      := $(datarootdir)
+mandir       := $(datarootdir)/man
+man1dir      := $(mandir)/man1
+man5dir      := $(mandir)/man5
+man7dir      := $(mandir)/man7
+srcdir       := .
+
+RINK_PATH    := $(prefix)/share/rink
+export RINK_PATH
+
+all: bin man
+
+fetch:
+	$(CARGO) fetch $(FETCHFLAGS)
+
+bin:
+	$(CARGO) build $(CARGOFLAGS) -p rink
+
+test:
+	$(CARGO) test $(CARGOFLAGS) --all
```

Comment:
You may not want to use `--release` while running tests , see the note here:
https://wiki.archlinux.org/title/Rust_package_guidelines#Check

Reply:
Thanks for catching that. Fixed here: https://github.com/tiffany352/rink-rs/commit/980fa25ba0560232d21ff81488fd7b43c44f20ee